src/McpBundle/Repository/McpScopeRepository.php

```
@@ -0,0 +1,50 @@
+    public function getScopeEntityByIdentifier(string $identifier): ?ScopeEntityInterface
+    {
+        if (! \in_array($identifier, McpScope::values(), true)) {
+            return null;
+        }
+
+        return new ScopeEntity($identifier);
+    }
+
+    public function finalizeScopes(
+        array $scopes,
+        string $grantType,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$grantType'. (undefined)

(UnusedFormalParameter)